src/VisualStudio/Core/Def/Implementation/GenerateType/GenerateTypeDialogViewModel.cs

```diff
@@ -319,7 +299,11 @@ internal bool TrySubmit()
                 if (lastIndexOfSeparatorInFullPath != -1)
                 {
                     var fileNameInFullPathInContainers = this.FullFilePath.Split(new[] { '\\' }, StringSplitOptions.RemoveEmptyEntries);
-                    this.FullFilePath = string.Join("\\", fileNameInFullPathInContainers.Select(str => str.TrimStart()));
+
+                    // Trim spaces of each component of the file name.
+                    // Note that path normalization changed between 4.6.1 and 4.6.2 and GetFullPath no longer trims trailing spaces.
```

Comment:
So we're already broken on machines that were upgraded? Do we not have tests for this?

Reply:
We have been broken if our dll was loaded to an .exe that was built against net462+. We are now using xunit runner built against net472, so that's where it showed up.